tree/dataframe/inc/ROOT/RDF/RLoopManager.hxx

```diff
@@ -161,7 +161,7 @@ public:
    RLoopManager(TTree *tree, const ColumnNames_t &defaultBranches);
    RLoopManager(ULong64_t nEmptyEntries);
    RLoopManager(std::unique_ptr<RDataSource> ds, const ColumnNames_t &defaultBranches);
-   RLoopManager(ROOT::RDF::RDatasetSpec spec);
+   RLoopManager(const ROOT::RDF::RDatasetSpec &spec);
```

Comment:
I think this can be `RDatasetSpec &&` since you just take it by moving it from the RDF constructor